### PR TITLE
Realistic examples in system tests

### PR DESCRIPTION
**Pull request by gilesknap**

### CODE REVIEW

**Comment by gilesknap** [comment]
@GDYendell you might be interested in this as it hopefully fixes an issue you raised with the ibek system tests.
But don't spend lots of time on it - the tests pass and the coverage is good (still need to add tests for the the subcommands that call shell - thats next).

**Comment by gilesknap** [comment]
@coretl you might be interested to know I have added checks for jinja escaping. Does this cover what you asked me about recently?

https://github.com/epics-containers/ibek/blob/858582d8a4242548273e9c051514b1aaddf3fbe6/tests/samples/iocs/ibek-mo-ioc-01.yaml#L26-L32

https://github.com/epics-containers/ibek/blob/858582d8a4242548273e9c051514b1aaddf3fbe6/tests/samples/support/asyn.ibek.support.yaml#L110-L121

https://github.com/epics-containers/ibek/blob/858582d8a4242548273e9c051514b1aaddf3fbe6/tests/samples/support/motorSim.ibek.support.yaml#L34-L44

See the expected results in https://github.com/epics-containers/ibek/tree/dev/tests/samples/outputs

**Review by GDYendell — COMMENTED** [review]
This is great! Just one comment.

File: tests/samples/support/simple.pvi.device.yaml
[the remark this answers is not in the document]
Comment (gilesknap) [reply tests/samples/support/simple.pvi.device.yaml:1]: Good point. Will add that

**Pull request merged**